bundles/admin/admin-layereditor/view/AdminLayerForm/AdditionalTabPane/CapabilitiesUpdate.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Message, Button } from 'oskari-ui';
+import { Controller } from 'oskari-ui/util';
+import { Numeric } from '../Numeric';
+import styled from 'styled-components';
+
+const Link = styled('a')`
+    margin-left: 10px;
+`;
+const getCapabilitiesUrl = ({ url, type, version, password, username }) => {
+    return Oskari.urls.getRoute('GetWSCapabilities', {
+        url,
+        type,
+        version,
+        pw: password,
+        user: username
+    });
+};
+
+export const CapabilitiesUpdate = ({ layer, controller }) => (
+    <Fragment>
+        <Numeric
+            value={layer.capabilitiesUpdateRate}
+            messageKey='capabilities.updateRate'
+            infoKeys='capabilities.updateRateDesc'
+            suffix='s'
+            allowNegative={false}
+            allowZero={false}
+            onChange={value => controller.setCapabilitiesUpdateRate(value)}>
+            <Fragment>
+                <Button value={layer.name} onClick={() => controller.updateCapabilities()}>
+                    <Message messageKey='capabilities.update' />
+                </Button>
+                <Link href={getCapabilitiesUrl(layer)} target='capabilities'>
```

Comment:
This shows the parsed capabilities as JSON? I think we want to call GetLayerCapabilities instead to get the "raw" xml.